services/m3dbnode/config/bootstrap.go

```diff
@@ -89,15 +90,15 @@ func (bsc BootstrapConfiguration) New(
 	adminClient client.AdminClient,
 ) (bootstrap.ProcessProvider, error) {
 	var (
-		bs  bootstrap.BootstrapperProvider
-		err error
+		bs     bootstrap.BootstrapperProvider
+		err    error
+		iopts  = opts.InstrumentOptions()
+		rsopts = result.NewOptions().
+			SetInstrumentOptions(opts.InstrumentOptions()).
+			SetDatabaseBlockOptions(opts.DatabaseBlockOptions()).
+			SetSeriesCachePolicy(opts.SeriesCachePolicy()).
+			SetIndexMutableSegmentAllocator(index.NewDefaultMutableSegmentAllocator(iopts))
```

Comment:
This wasn't here before right? Why is it being added now

Reply:
need the ability to create segments for index bootstrap results (they include segments)

Reply:
As prateek mentions.